### ISSUE TITLE
Socket server did not execute the callback for setAttributes with data

### ISSUE THREAD

**Issue by Mihai-github** [tawk/tawk-messenger-react#27]
Hello, 

Sentry reported to us recently this error coming from the messenger widget we have integrated from you guys, is there something we need to worry about the below error?

```
Socket server did not execute the callback for setAttributes with data : {"name":"Name","email":"email@email.com","hash":"hash"}
```

**Comment by andrey690062** [comment]
Any update?

**Comment by gutuJosh** [comment]
Hi Mihai! I have the same problem to! I see that the report is a bit old, have you managed to understand why is happening, in the meantime?

**Comment by MartinXPN** [comment]
I have the same issue as well. Not sure why it's happening...

**Comment by Red-Killer** [comment]
Make sure that you added a error callback this fixed it for me
![image](https://github.com/tawk/tawk-messenger-react/assets/59504268/62c77a2c-7327-475e-84af-d24ea87d97c8)

**Comment by jaoaustero** [comment]
Hi is this still happening? we release a lot of changes before we check this issue

**Comment by wnasich** [comment]
yes, it is still happening. Below screenshots from JS error tracking from a small web app with a few of active users each day:

![Selection_2130](https://github.com/user-attachments/assets/464f2a8a-ce6d-40fc-977c-50de16606926)

![Selection_2131](https://github.com/user-attachments/assets/8337f985-b22f-4fd8-ae95-48dd0b9e5a41)

Thanks @jaoaustero

**Comment by jaoaustero** [comment]
@wnasich thanks for providing a screenshots, I'm not familiar with the sentry. But our setAttributes API returned `undefined` when it was success because the callback was intended to log if theres error

I will inform this on our team, but if sentry returning this error

did you observe if your data is set in the dashboard?

**Comment by pedro-brinfer-antunes** [comment]
Same here!
Any update? @jaoaustero

**Comment by Bumboobee** [comment]
The error still are happening in 2025, its been two years since the issue was open and there is no update at all? Please @jaoaustero how can we fix this? 

Sentry keep warning our team about this issue.
